### PR TITLE
[4.1.0] | Release notes 4.1.0

### PR DESCRIPTION
**Pull request by Kaur-Parminder**
Release notes 4.1.0

### CODE REVIEW

File: release-notes/4.1/4.1.0.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+### Use `Attestation Protocol = None` to skip enclave attestation for `VBS` enclaves.
+
+A new attestation protocol called `None` will be allowed in the connection string. This protocol will allow users to forgo enclave attestation for `VBS` enclaves. When this protocol is set, the enclave attestation URL property is optional.  
+
+Connection string examples:
+
+```cs
+//Attestation protocol NONE with no URL
+"Data Source = {server}; Initial Catalog = {db}; Column Encryption Setting = Enabled; Attestation Protocol = None;"
+//Attestation protocol NONE with URL
```
Comment (johnnypham) [inline release-notes/4.1/4.1.0.md:20]: I would remove this second connection string, just to emphasize that None should not be used with a URL.

File: release-notes/4.1/4.1.0.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+### Added
+
+- Added new Attestation Protocol `None` for `VBS` enclave types. This protocol will allow users to forgo enclave attestation for VBS enclaves. [#1489](https://github.com/dotnet/SqlClient/pull/1489) [Read more](#Use-Attestation-Protocol-=-None-to-skip-enclave-attestation-for-VBS-enclaves)
+
+### Use `Attestation Protocol = None` to skip enclave attestation for `VBS` enclaves.
+
+A new attestation protocol called `None` will be allowed in the connection string. This protocol will allow users to forgo enclave attestation for `VBS` enclaves. When this protocol is set, the enclave attestation URL property is optional.  
+
+Connection string examples:
```
Comment (johnnypham) [inline release-notes/4.1/4.1.0.md:15]:
```suggestion
Connection string example:
```

**Review by johnnypham — CHANGES_REQUESTED** [review]

File: release-notes/4.1/4.1.0.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
+# Release Notes
+
+## Microsoft.Data.SqlClient 4.1.0 released 28 January 2022
+
+This update brings the below changes over the previous preview release:
+
+### Added
+
+- Added new Attestation Protocol `None` for `VBS` enclave types. This protocol will allow users to forgo enclave attestation for VBS enclaves. [#1489](https://github.com/dotnet/SqlClient/pull/1489) [Read more](#Use-Attestation-Protocol-=-None-to-skip-enclave-attestation-for-VBS-enclaves)
```
Comment (DavoudEshtehari) [inline release-notes/4.1/4.1.0.md:9]: The `Read more` link is broken cause of using the special character "`". I'd suggest using a simpler title.

**Review by DavoudEshtehari — CHANGES_REQUESTED** [review]
Missing update the `CHANGELOG.md`.

File: release-notes/4.1/4.1.0.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
+# Release Notes
+
+## Microsoft.Data.SqlClient 4.1.0 released 28 January 2022
```
Comment (Kaur-Parminder) [inline release-notes/4.1/4.1.0.md:3]:
```suggestion
## Microsoft.Data.SqlClient 4.1.0 released 31 January 2022
```

File: release-notes/4.1/4.1.0.md (hunk 3 above)
Comment (Kaur-Parminder) [inline release-notes/4.1/4.1.0.md:9]:
```suggestion
- Added new Attestation Protocol `None` for `VBS` enclave types. This protocol will allow users to forgo enclave attestation for VBS enclaves. [#1489](https://github.com/dotnet/SqlClient/pull/1489) [Read more](##use-attestation-protocol-none-to-skip-enclave-attestation-for-vbs-enclaves)
```

File: release-notes/4.1/4.1.0.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+This update brings the below changes over the previous preview release:
+
+### Added
+
+- Added new Attestation Protocol `None` for `VBS` enclave types. This protocol will allow users to forgo enclave attestation for VBS enclaves. [#1489](https://github.com/dotnet/SqlClient/pull/1489) [Read more](#Use-Attestation-Protocol-=-None-to-skip-enclave-attestation-for-VBS-enclaves)
+
+### Use `Attestation Protocol = None` to skip enclave attestation for `VBS` enclaves.
```
Comment (Kaur-Parminder) [inline release-notes/4.1/4.1.0.md:11]:
```suggestion
### Use Attestation Protocol None to skip enclave attestation for VBS enclaves
```

**Comment by Kaur-Parminder** [comment]
> Missing update the `CHANGELOG.md`.

Added

File: release-notes/4.1/4.1.0.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+## Microsoft.Data.SqlClient 4.1.0 released 31 January 2022
+
+This update brings the below changes over the previous preview release:
+
+### Added
+
+- Added new Attestation Protocol `None` for `VBS` enclave types. This protocol will allow users to forgo enclave attestation for VBS enclaves. [#1489](https://github.com/dotnet/SqlClient/pull/1489) [Read more](#use-attestation-protocol-none-to-skip-enclave-attestation-for-vbs-enclaves)
+
+### Use Attestation Protocol None to skip enclave attestation for VBS enclaves
```
Comment (DavoudEshtehari) [inline release-notes/4.1/4.1.0.md:11]: NIT: Please update the #1488 accordingly.
```suggestion
### Introduce Attestation Protocol None
```

File: release-notes/4.1/4.1.0.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
+# Release Notes
+
+## Microsoft.Data.SqlClient 4.1.0 released 31 January 2022
+
+This update brings the below changes over the previous preview release:
+
+### Added
+
+- Added new Attestation Protocol `None` for `VBS` enclave types. This protocol will allow users to forgo enclave attestation for VBS enclaves. [#1489](https://github.com/dotnet/SqlClient/pull/1489) [Read more](#use-attestation-protocol-none-to-skip-enclave-attestation-for-vbs-enclaves)
```
Comment (Kaur-Parminder) [inline release-notes/4.1/4.1.0.md:9]:
```suggestion
- Added new Attestation Protocol `None` for `VBS` enclave types. This protocol will allow users to forgo enclave attestation for VBS enclaves. [#1489](https://github.com/dotnet/SqlClient/pull/1489) [Read more](#introduce-attestation-protocol-none)
```

File: release-notes/4.1/4.1.md [hunk 8]
```diff
@@ -0,0 +1,7 @@
+# Microsoft.Data.SqlClient 4.1 Releases
+
+The following Microsoft.Data.SqlClient 4.1 stable releases have been shipped:
+
+| Release Date | Version | Notes |
+| :-- | :-- | :--: |
+| 2022/01/28 | 4.1.0 | [release notes](4.1.0.md) |
```
Comment (DavoudEshtehari) [inline release-notes/4.1/4.1.md:7]:
```suggestion
| 2022/01/31| 4.1.0 | [release notes](4.1.0.md) |
```

File: release-notes/4.1/README.md [hunk 9]
```diff
@@ -0,0 +1,7 @@
+# Microsoft.Data.SqlClient 4.1 Releases
+
+The following Microsoft.Data.SqlClient 4.1 stable releases have been shipped:
+
+| Release Date | Version | Notes |
+| :-- | :-- | :--: |
+| 2022/01/28 | 4.1.0 | [release notes](4.1.0.md) |
```
Comment (DavoudEshtehari) [inline release-notes/4.1/README.md:7]:
```suggestion
| 2022/01/31 | 4.1.0 | [release notes](4.1.0.md) |
```

**Review by DavoudEshtehari — APPROVED** [review]

**Review by johnnypham — APPROVED** [review]

**Review by JRahnama — APPROVED** [review]

**Pull request merged**